mslib/mswms/mpl_vsec_styles.py

```diff
@@ -114,6 +114,7 @@ class VS_GenericStyle(AbstractVerticalSectionStyle):
     """
     Vertical section of chemical species/other stuff
     """
+    name = "VS_GenericStyle"
```

Comment:
I do not understand the purpose of this.
This class should not be used per se, only classes derived from this, which get assigned there own name.

How does this solution connect to issue 1423?

The (new) assert should discover mis-configurations, where multiple identical layers are configured, which, previously, would choose the last one to be used.

The error should only be generated for faulty configuration...?

Reply:
hmm, propabably.

Reply:
found the cause for this.